packages/frontend/components/Borrow/LTVProgressBar.tsx

```diff
@@ -32,21 +33,64 @@ export default function LTVProgressBar(props: LTVProgressBarProps) {
               sx={{
                 marginRight: "0.313rem",
                 fontSize: "0.875rem",
+                display: { xs: "none", sm: "inline" },
               }}
             />
           </Tooltip>
-          <Typography variant="xsmall">45% LTV (Recommended)</Typography>
+          <ClickableTooltip title="???" placement="top">
+            <InfoOutlinedIcon
+              sx={{
+                marginRight: "0.313rem",
+                fontSize: "0.875rem",
+              }}
+            />
+          </ClickableTooltip>
+          <Typography
+            variant="xsmall"
+            sx={{ display: { xs: "none", sm: "inline" } }}
+          >
+            45% LTV (Recommended)
+          </Typography>
+
+          <Typography
+            variant="xsmall"
+            sx={{ display: { xs: "inline", sm: "none" } }}
+          >
+            SAFE LTV: 45%
+          </Typography>
         </Grid>
         <Grid item alignItems="center" marginRight="3rem">
-          <Typography variant="xsmall">75% LTV (MAX)</Typography>
           <Tooltip title="???" placement="top">
             <InfoOutlinedIcon
               sx={{
-                marginLeft: "0.313rem",
+                marginRight: "0.313rem",
                 fontSize: "0.875rem",
+                display: { xs: "none", sm: "inline" },
```

Comment:
Open question: do we need to specify `sm: inline` ? I think this is the default 🤔  Same remark for the 2 `Typography` below